Src/SpriteFont.cpp

```diff
@@ -478,10 +478,13 @@ XMVECTOR XM_CALLCONV SpriteFont::MeasureString(_In_z_ wchar_t const* text, bool
     {
         UNREFERENCED_PARAMETER(advance);
 
+        auto isWhitespace = iswspace(wchar_t(glyph->Character));
```

Comment:
For MeasureString, you only have one instance of checking ``iswspace`` so catching it in a variable doesn't seem needed. Makes sense for the MeasureDrawBounds which uses it twice.

Reply:
I'll deal with it as additional code review.

Reply:
Fixed in [this commit](https://github.com/microsoft/DirectXTK12/commit/0b2f50b125a814bfbe487e462e958a75492629d4)